setup.cfg

```diff
@@ -32,7 +32,9 @@ install_requires =
 	ansible-runner
 	websockets
 	drools_jpy == 0.3.8
-    watchdog
+	watchdog
+	psycopg[c]
```

Comment:
I am a bit concerned that we are going to run into issues with lots of dependencies in ansible-rulebook.     This dependency is fine, but any more actions with dependencies should go into action plugins.

Reply:
We already have problems with ansible.eda dependencies on different platforms.   We might need platform specific dependencies.

Reply:
psycopg[c] requires a compiler for installation.

Reply:
The dockerfile already includes the compiler and for downstream the change is also already being coordinated.

Reply:
As it is now everything is in the image, but there is a parallel effort to improve the dockerfile with multistage builds:
https://github.com/ansible/ansible-rulebook/pull/639
So, after this change, that PR can be update to just use the compiler in the build stage and not ship it in the final one.